### ISSUE TITLE
Add GSSAPI auth and host-based auth configuration to security overview doc

### ISSUE THREAD

**Issue by rolandcrosby** [cockroachdb/docs#4656]
The [table of security features](https://www.cockroachlabs.com/docs/dev/security-overview.html#security-features-in-cockroachdb) on the security overview page should be updated for 19.1, now that we have GSSAPI authentication and configurable authentication methods by host.

**Comment by jseldess** [comment]
Good point, @rolandcrosby. Do you think GSSAPI should be mentioned somehow in the body of that page as well?

**Comment by jseldess** [comment]
Good point, @rolandcrosby. Do you think GSSAPI should be mentioned somehow in the body of that page as well?
